config/maps.txt

```diff
@@ -41,7 +41,9 @@ endmap
 map kilostation
 	votable
 endmap
+
 map limastation
+
```

Comment:
this will need to have votable once we are fully ready

Reply:
I'm aware, but for now it's intentionally non-votable